docker/php/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM php:7.2-fpm-alpine
+
+# Install Symfony requirements
+RUN \
+    apk add --no-cache --virtual .build-deps \
+        git \
+        icu-dev \
+        yarn && \
+    docker-php-ext-install \
+        intl \
+        opcache
+
+# Install Composer globally
+ENV COMPOSER_ALLOW_SUPERUSER 1
+RUN \
+    curl -sS https://getcomposer.org/installer | php && \
+    mv composer.phar /usr/local/bin/composer && \
+    composer global require "hirak/prestissimo:dev-master" --no-suggest --optimize-autoloader --classmap-authoritative
```

Comment:
not needed IMO. Flex already optimizes the downloading of packages to do it in parallel (and it might now play well with `hirak/prestissimo` if they both try to do it)

Reply:
I made several tests on my local environment and Composer is still faster with `hirak/prestissimo`. You will find the results below. Did I miss something?

```
# without hirak/prestissimo
make uninstall ; rm -rf vendor/ ; time make build start composer
[...]
make build start composer  1,25s user 0,51s system 1% cpu 1:33,85 total
```

```
# with hirak/prestissimo
make uninstall ; rm -rf vendor/ ; time make build start composer
[...]
make build start composer  1,25s user 0,49s system 3% cpu 49,488 total
```

And I'm not sure, but it seems for me that the incompatibility between `symfony/flex` and `hirak/prestissimo` was fixed with https://github.com/symfony/flex/issues/381.